setup.py

```diff
@@ -134,6 +134,6 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     long_description=LONG_DESCRIPTION,
     platforms=['any'],
     classifiers=CLASSIFIERS,
-    install_requires=['boto>=2.1.0'],
+    install_requires=['boto>=2.31.0'],
```

Comment:
I was trying to figure out what boto version begins support for python 3... do you have a link handy?